controller/configurations/amazon_prime.py

```diff
@@ -0,0 +1,91 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import traceback as tb
+import utils.cookie_manager as cm
+
+class AmazonPrimeVideo:
+    ''' Amazon Prime Video configuration '''
+
+    def __init__(self):
```

Comment:
Username & Password arguments? How are these passed onto the class attributes of self.username & self.password

Reply:
Password Manager is still under development. Till now I was directly using hardcoded strings for testing purpose(obviously can't commit with that). Later the functions will use the Password Manager to retrieve and set self.username and self.password when required.